charts/latest/csi-driver-nfs/templates/snapshotclass.yaml

```diff
@@ -0,0 +1,9 @@
+{{- if .Values.externalSnapshotter.create }}
```

Comment:
pls als create a snapshotclass.yaml under deploy folder

Reply:
Alright will do that

Reply:
Done